releases/release-5.1.md

```diff
@@ -0,0 +1,318 @@
+---
+title: TiDB 5.1 Release Notes
+---
+
+# TiDB 5.1 Release Notes
+
+发版日期：2021 年 6 月 24 日
+
+TiDB 版本：5.1
+
+在 5.1 版本中，你可以获得以下关键特性：
+
+- 支持 MySQL 8 中的公共表表达式 (Common Table Expression)，提高了 SQL 语句的可读性与执行效率。
+- 支持 MySQL 8 中的动态权限（Dynamic Privileges）配置，实现对某些操作更细粒度的控制。
+- 支持通过 Stale Read 功能直接读取本地副本数据，降低读取延迟，提升查询性能。
+- 新增锁视图 (Lock View) 功能方便 DBA 观察事务加锁情况以及排查死锁问题（实验特性）。
+- 新增 TiKV 后台任务写入限制（TiKV Write Rate Limiter)，保证读写请求的延迟稳定性。
+
+## 兼容性更改
+
+### 系统变量
+
+| 变量名   | 修改类型   | 描述   |
+|:----------|:-----------|:-----------|
+| `tidb_enable_enhanced_security`  | 新增 | 表示所连接的 TiDB 服务器是否启用了安全增强模式（SEM），在不重新启动 TiDB 服务器的情况下不能改变该变量。 |
+| `cte_max_recursion_depth`  | 新增 | 用于控制公共表表达式最大递归深度。 |
+| `init_connect`  | 新增 | 用于控制初始连接。 |
+| `tidb_analyze_version`  | 修改 | 用于控制所收集到的统计信息。默认值从 `1` 修改为 `2`，默认作为实验特性启用。 |
+| `tidb_enforce_mpp`  | 新增 | 用于忽略优化器代价估算，强制使用 mpp 模式。bool 类型，默认值为 false。 |
+
+### 配置文件参数
```

Comment:
tikv also add [resource-metering],cdc.sink-memory-quota;
tikv deleted rocksdb.ver-defaultcf, but may be these param has not been published to customer

Reply:
pd deleted "trace-region-flow" param, and add a new param :pd-server.flow-round-by-digit

Reply:
@nolouch @yiwu-arbug @tabokie Please take a look whether the configs mentioned by @seiya-annie are exposed to users and should be documented here. Thanks!

Reply:
> tikv also add [resource-metering],cdc.sink-memory-quota;
> tikv deleted rocksdb.ver-defaultcf, but may be these param has not been published to customer

Could you elaborate `resource-metering`?

Reply:
@seiya-annie PTAL